components/Common/Footer.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable react/jsx-no-undef */
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable react/jsx-no-comment-textnodes */
+/* eslint-disable max-len */
+/* eslint react/jsx-filename-extension: 0 */
+/* eslint react/jsx-no-literals: 0 */
+/* eslint arrow-body-style: 0 */
+// eslint-disable-next-line react/jsx-no-comment-textnodes
+// eslint-disable-next-line react/jsx-no-undef
+// eslint-disable-next-line jsx-a11y/anchor-is-valid
+
```

Comment:
go through this file and save all strings that is not a `className` to the `constants.js` file